### ISSUE TITLE
Support text selection in running mode

### ISSUE THREAD

**Issue by harry75369** [verygoodgraphics/vgg_runtime#381]
### Detailed description

Text selection is one of accessibility problems that should be prioritized.

### Possible implementation

It depends.

If only web platform is going to be supported, there is a much easier way to use overridden DOM node.

If not, a serious of work amount is expected in the C++ engine.
